tests/ci/codebuild/release/al2_aarch64_test.yml

```diff
@@ -0,0 +1,62 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+version: 0.2
+
+env:
+  variables:
+    GO_TAG: go1.18.3
+phases:
+  # TODO: Everything in the install section should be moved into a prebuilt image
+  install:
+    runtime-versions:
+      java: corretto11
+    commands:
+      - yum -y update
+      - yum -y group install 'Development Tools'
+      # - curl -L -O https://github.com/Kitware/CMake/releases/download/v3.24.0-rc2/cmake-3.24.0-rc2.tar.gz
+      - curl -L -O https://downloads.sourceforge.net/ltp/lcov-1.14-1.noarch.rpm
```

Comment:
do we do anything with this RPM after it's downloaded?

also, minor nit: delete commented lines

Reply:
This should be installed. I'll readd that.